vms/platformvm/utxo/camino_locked.go

```diff
@@ -210,28 +213,27 @@ func (h *handler) Lock(
 	[]*avax.TransferableInput, // inputs
 	[]*avax.TransferableOutput, // outputs
 	[][]*crypto.PrivateKeySECP256K1R, // signers
+	[]*secp256k1fx.OutputOwners, // owners
 	error,
 ) {
 	switch appliedLockState {
 	case locked.StateBonded,
 		locked.StateDeposited,
 		locked.StateUnlocked:
 	default:
-		return nil, nil, nil, errInvalidTargetLockState
+		return nil, nil, nil, nil, errInvalidTargetLockState
 	}
 
-	addrs := set.NewSet[ids.ShortID](len(keys)) // The addresses controlled by [keys]
-	for _, key := range keys {
-		addrs.Add(key.PublicKey().Address())
-	}
+	addrs, signer := secp256k1fx.SplitPrivateKeys(keys)
```

Comment:
```suggestion
	spendFromAddrs, signer := secp256k1fx.SplitPrivateKeys(keys)
```
not sure, but imo would give better explanation about splitting